### DISCUSSION TITLE
Cannot change background color of PopoverArrow with properties like backgroundColor.

### DISCUSSION THREAD

**Discussion by surajpheudin** [chakra-ui/chakra-ui#9290] [General] [answered]
Description
The background color of PopoverContent can be changed easily with properties like `backgroundColor="red"` but to change the background color of PopoverArrow, we have to change the value of the variable directly.
e.g
```
css={{
            "--arrow-background": "red",
          }}
```

Problem Statement/Justification
We cannot change the background color of PopoverArrow using CSS properties like `backgroundColor`, `color` or `borderColor`.

Proposed Solution or API
1. It would be great to have PopoverArrow background color changed just by using` backgroundColor="somecolor"` property 
or, 
2. This solution can be mention in the docs,
```
css={{
            "--arrow-background": "red",
          }}
```

Alternatives
No response

Additional Information
No response

**Comment by segunadebayo** [comment, marked as answer]
Here you go:
https://github.com/chakra-ui/chakra-ui/blob/main/apps/compositions/src/examples/popover-with-custom-bg.tsx

Hope that helps

**Reply by surajpheudin** [reply]
Yes, thank you.

**Reply by surajpheudin** [reply]
Also, I noticed.
Apply `css={{ "--popover-bg": "somecolor" }}` as mentioned in docs, does not change the borderColor of PopoverArrowTip.

![image](https://github.com/user-attachments/assets/dd180f62-7ded-43d8-811b-297069ec19a1)

It is quite visible if we have different backgrounds, I couldn't find a way to change the PopoverArrowTip borderColor without
1. Adding slotRecipe for popover in theme and changing color inside **arrowTip** , or
2. Directly targeting _chakra-popover__arrowTip_ class and applying borderColor in it.

Is it supposed to be like this?
